psycopg/replication_cursor_type.c

```diff
@@ -138,7 +152,9 @@ consume_stream(replicationCursorObject *self,
     CLEARPGRES(curs->pgres);
 
     self->consuming = 1;
-    set_status_interval(self, keepalive_interval);
+    if (keepalive_interval >= 1) {
```

Comment:
probably better as `if (keepalive_interval > 0.0)` because being it >1 sec is a "business rule" but being it nonzero is "the user specified something" (which passed validation).